src/main/java/org/jabref/gui/externalfiletype/CustomizeExternalFileTypesViewModel.java

```diff
@@ -34,23 +35,24 @@ public void resetToDefaults() {
     public void addNewType() {
         CustomExternalFileType type = new CustomExternalFileType("", "", "", "", "new", IconTheme.JabRefIcons.FILE);
         fileTypes.add(type);
-        edit(type);
+        edit(type, true);
     }
 
     public ObservableList<ExternalFileType> getFileTypes() {
         return fileTypes;
     }
 
-    public void edit(ExternalFileType type) {
+    public void edit(ExternalFileType type, boolean add) {
```

Comment:
please add a public `add(ExternalFileType)` method and make this one here private (I'm not a big fan of these boolean toggles that completely alter the nature of the method).

Reply:
I found a better solution.